polaris-react/src/components/Text/Text.tsx

```diff
@@ -58,6 +60,8 @@ export interface TextProps {
   variant?: Variant;
   /** Visually hide the text */
   visuallyHidden?: boolean;
+  /** Add a underline or line-through style to the text */
+  decoration?: Decoration;
```

Comment:
Is that enough to get reflected in the polaris documentation website?

Reply:
Ohh I'm not sure, maybe the readme is the one used, good point! I'll double check!

Reply:
To get text decoration examples to show up in the style guide, you'll need to add a new example file to the `polaris.shopify.com/pages/examples` dir (here is an example for [text-weight](https://github.com/Shopify/polaris/blob/main/polaris.shopify.com/pages/examples/text-weight.tsx)) and then reference the new file in the `polaris.shopify.com/content/components/typography/text.md` examples metadata on line 22. You can run the styleguide locally by using `yarn dev` in the `polaris.shopify.com` directory to test.

Reply:
Thanks, makes sense. I'll apply these changes and re-request review :)

Reply:
Ahh thanks, it had failed and I didn't notice, but all good now :) Re-review requested